scripts/validate

```diff
@@ -4,7 +4,7 @@ set -e
 echo Running: flake8 python linting
 
 cd $(dirname $0)/../tests/integration
-tox -e flake8
+#tox -e flake8
```

Comment:
Does this need to go? Just remove it and add why

Reply:
Oh good catch, thank you. I didn't mean to remove it, I just commented it out so it wouldn't run while I was testing locally. I'll undo this change.

Reply:
@superseb Fixed. Ty